src/main/resources/config/systemMMn/OUTPUTS_DEFAULT.json

```diff
@@ -3,7 +3,7 @@
 	"name": "systemMMn",
 	"values": [
 		{
-			"value": "Ro,US,a,P0,Pn,PNc,NAvg,D2N,QAvg,D2Q,WAvg,D2W,PiWr,PiW90,PiW95,WAvgW,EWW0,D2WW0,C1,C2,TAvg,ET2,D2T,PiT90,PiT95,FTt,FWt,PNn"
+			"value": "Ro,P0,US,a,Pn,PNc,PNn,NAvg,D2N,QAvg,D2Q,WAvg,D2W,PiWr,PiW90,PiW95,WAvgW,EWW0,D2WW0,FWt,TAvg,ET2,D2T,FTt,PiT90,PiT95"
```

Comment:
Please remove the C1 and C2 from the calculator also (after this PR of course :) )